xml/System.Windows/ThemeModeConverter.xml

```diff
@@ -61,10 +63,10 @@
         <Parameter Name="sourceType" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="typeDescriptorContext">To be added.</param>
-        <param name="sourceType">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="typeDescriptorContext">The ITypeDescriptorContext for this call.</param>
+        <param name="sourceType">The Type being queried for support.</param>
+        <summary>CanConvertFrom - Returns whether or not this class can convert from a given type.</summary>
+        <returns>bool - <see langword="true" /> if thie converter can convert from the provided type, <see langword="false" /> if not.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" /> if the converter can convert from the provided type; otherwise, <see langword="false" />.</returns>
```